File: src/Backend/Portal.Domain/Entidade/Tarefa.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace Portal.Domain.Entidade;

[Table("Tarefas")]
public class Tarefa :EntidadeBase
{
    public string Titulo { get; set; }
    public string Descricao { get; set; }
    public string DataConclusao { get; set; }
    public long UsuarioId { get; set; }
    public Usuario Usuario { get; set; }
}

File: src/Frontend/src/app/model/Tarefa.ts
import { RespostaUsuarioJson } from "./RespostaUsuarioJson";

export interface Tarefa {
    id: number;
    titulo: string;
    descricao: string;
    dataCriacao: Date;
    dataConclusao?: string;
    usuarioId: number;
    usuario: RespostaUsuarioJson;
}
